okhttp/src/main/resources/META-INF/services/com.lightstep.tracer.shared.CollectorClientProvider

```diff
@@ -0,0 +1 @@
+com.lightstep.tracer.shared.HttpCollectorClientProvider
```

Comment:
not really sure what this is for?

Reply:
...I mostly just copy-pasta'd here but the underlying definition is something to do with the `ServiceLoader`. Basically the file name is the interface or abstract class you are implementing and the contents are all of the classes that implement that interface that you wish to register (`\n` separated).